.github/workflows/manylinux_2014.yml

```diff
@@ -170,7 +170,16 @@ jobs:
             ${{ fromJSON(needs.docker.outputs.images).ov_build.manylinux2014_x86_64 }} \
             /bin/bash -c "
               python3.12 -m pip install -r /work/src/src/bindings/python/wheel/requirements-dev.txt
```

Comment:
BTW, if we don't build wheel, we don't need to install `src/bindings/python/wheel/requirements-dev.txt`